module_utils/network_lsr/nm/provider.py

```diff
@@ -65,6 +65,18 @@ def get_client_version(self):
         nm_client = client.get_client()
         return nm_client.get_version()
 
+    def nm_support_version_cmp(self, nm_support_version):
+        v = tuple(self.get_client_version().split("."))
```

Comment:
Please do not introduce new single letter variables.

Reply:
I replaced with meaningful variable, thanks.